project/.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
```

Comment:
Sorry but I'm simply not sold on pre-commit :sweat_smile: 
I find it slow and redundant: it creates virtualenvs and installs tools in a cache somewhere, instead of reusing the one I already dev-depend on through Poetry.

I could consider adding support for it through a Copier option though, which would default to false. If the value is false, this file would then be deleted by the post generation hook.

If we go this way, note that I won't maintain it myself!

Reply:
I can't argue with that. :) 

Optional but not enabled by default seems like a safe bet. 

I will say that it takes a bit to figure out and setup and its handy to know my stuff is passing before i push to CI which if i did too much could cost me money after too many times. 

Definitely don't expect you to maintain it but figured others might find it handy and can optionally enable it easily if desired.